Other/MergeShops/Adam1a1Merge.cs

```diff
@@ -0,0 +1,152 @@
+
+                case "Fresh Ectoplasm":
+                case "IOU Slip":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    Core.RegisterQuests(8009);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("djinn", "Lamia", "Tasty Poison", 10, log: false);
+                        Core.HuntMonster("vendorbooths", "r10", "Left", "*", "Caffeine Imp", "Coffee Beans", 10, log: false);
+                        Core.HuntMonster("charredpath", "r7", "Right", "*", "Toxic Wisteria", "Necessary Antidote", 1, log: false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "EctoBlade":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("shadowfallwar", "garden1", "bottom", "*", "Skeletal Fire Mage", "Ectoblade", 1, log: false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Fresh Ectoplasm":
+                case "IOU Slip":
+                case "Ectoplasmic Chains":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    Core.RegisterQuests(8010);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("doomwood", "r2", "Right", "*", "Doomwood Ectomancer", "Refined Ectoplasm", 10, log: false);
+                        Core.HuntMonster("ectocave", "Boss", "Left", "*", "Ektorax", "Ektorax's Ectoplasm", 1, log: false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Reho's Golden Sword Hilt":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    Core.RegisterQuests(8011);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("extinction", "r1", "Left", "Control Panel", "Iron II.0" 4, log: false);
+                        Core.HuntMonster("doomwood", "r5", "Bottom", "Doomwood Treeant", "Wood", 10, log: false);
+                        Core.HuntMonster("crashsite", "Dwakel Blaster", "Big Iron Bolts", 10, log: false);
+                        Core.findIngredients("portalmaze", "r3", "Left", "Piece of Cake", 1, log: false);
```

Comment:
as from discord, this is more then likely you trying todo  a Core.Getmapitem.
you can find the mapitemid from going to the map, and opening 
tools > grabber > scroll to "Get Mapitem IDs", and clicking grab once on the map. the ids will show up beside the item along with its associated QuestID. some times the mapitemid wont exist, at that point you'll have to force accept the Quest, and open  Packet > logger > enable (disable combat & rest packets in options > game), and go through till u get the Mapitemid packet and get it from there. (Ik that can be annoying :P)